package.json

```diff
@@ -53,6 +53,11 @@
     "test:unit": "yarn test 'test/unit/**/*.spec.ts'",
     "test:unit:parallel": "yarn test:parallel 'test/unit/**/*.spec.ts'"
   },
+  "resolutions": {
+    "cli-table3@^0.5.0/colors": "1.4.0",
+    "cli-table@^0.3.1/colors": "1.0.3",
+    "eth-gas-reporter/colors": "1.4.0"
```

Comment:
do we need this?

Reply:
Yeap. If not, node module installation breaks. Fixed in PR #35 .